packages/foreigntypes/posixtypes.pony

```diff
@@ -0,0 +1,95 @@
+// This file defines the standard Posix type to make FFI calls easier
+// to manage and more consistent.
+
+type BlkCnt is I64
+
+// not consistent across OSX (always 32 bit int)/Linux (Long)
+type BlkSize is ILong
+type OSXBlkSize is I32
```

Comment:
@jemc What you suggested is how I initially tried it and the compiler complained.

I think we're likely going to have to merge as is until Pony's type checker is enhanced as @sylvanc mentioned.